libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/NSUserDefaults+SFAdditions.h

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
Update all of these to use the standard Mobile SDK header/copyright from one of the other files.